reco_utils/recommender/deeprec/deeprec_utils.py

```diff
@@ -389,6 +388,14 @@ def cal_metric(labels, preds, metrics):
             pred[pred < 0.5] = 0
             f1 = f1_score(np.asarray(labels), pred)
             res["f1"] = round(f1, 4)
+        elif metric == 'mae':
```

Comment:
we had a FIXME at the beginning of this function to make it explicit. We also have an issue to do the same in the LigthGBM notebook https://github.com/microsoft/recommenders/issues/762

do you think you could refactor this?